### PR TITLE
Angular: patched for readily usable engine

### PR DESCRIPTION
**Pull request by hollowaykeanho**
It appears the existing Angular engine is a work-in-progress one. Hence, let's update it to generate a SEO-capable and search friendly static website seamlessly. Hence, let's do this.

This patch upgrades the Angular engine to a readily usable engine in Angular/ directory.

### CODE REVIEW

File: Angular/.ci/test_unix-any.sh [hunk 1]
```diff
@@ -44,13 +45,13 @@ if [ $(OS_Is_Run_Simulated) -eq 0 ]; then
         I18N_Simulate_Testing
         return 0
 else
-        ___browser="$(type -p google-chrome)"
+        ___browser="$(type -p chromium)"
```
Comment (corygalyna) [inline Angular/.ci/test_unix-any.sh]: No reason to change this. Revert it.

**Review by corygalyna — CHANGES_REQUESTED** [review]
Completed review. There is 1 problem. Please fix it.

**Review by hollowaykeanho — COMMENTED** [review]
scanned cleared. Please review again.

**Review by JeanShuralyov — APPROVED** [review]
Nice graphics! All cleared from my side.

File: Angular/.ci/test_unix-any.sh (hunk 1 above)
Comment (JeanShuralyov) [reply Angular/.ci/test_unix-any.sh]: No longer spotted in the new force-commit.

**Review by corygalyna — APPROVED** [review]
Done. All cleared!

File: Angular/.ci/test_unix-any.sh (hunk 1 above)
Comment (corygalyna) [reply Angular/.ci/test_unix-any.sh]: concur.

**Pull request merged**